auth/proxy/alarms.yaml

```
@@ -0,0 +1,242 @@
+          ReturnData: true
+      AlarmActions:
+        - !Ref AlarmTopic
+      OKActions:
+        - !Ref AlarmTopic
+      Threshold: 10
+      TreatMissingData: notBreaching
+      Tags:
+        - Key: Product
+          Value: GOV.UK
+        - Key: Environment
+          Value: !Ref Environment
+        - Key: System
+          Value: Authentication
+          
+  AttestationLowCompletionAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      ActionsEnabled: true
+      AlarmDescription: "A large proportion of Attestation requests have not completed successfully.\nSee resolution steps in runbook: \nhttps://gov-uk.atlassian.net/wiki/spaces/GOVUK/pages/4517658627/Runbook+High+number+of+attestation+failures\n"
+      AlarmName: !Sub ${AWS::StackName}-attestation-low-completion
+      ComparisonOperator: LessThanOrEqualToThreshold
+      DatapointsToAlarm: 1
+      EvaluationPeriods: 1
+      Metrics:
+        - Id: lambdaLogStarted
+          Label: Sum of ATTESTATION_STARTED messageCodes for lambda
+          MetricStat:
+            Metric:
+              Dimensions:
+                - Name: MessageCode
+                  Value: ATTESTATION_STARTED
+                - Name: Level
+                  Value: INFO
+              MetricName: ClientAttestationMessageCode
+              Namespace: !Sub ${AWS::StackName}/LogMessages
+            Period: 60
+            Stat: Sum
+          ReturnData: false
+        - Id: lambdaLogCompleted
+          Label: Sum of ATTESTATION_COMPLETED messageCodes for lambda
+          MetricStat:
+            Metric:
+              Dimensions:
+                - Name: MessageCode
+                  Value: ATTESTATION_COMPLETED
+                - Name: Level
+                  Value: INFO
+              MetricName: ClientAttestationMessageCode
+              Namespace: !Sub ${AWS::StackName}/LogMessages
+            Period: 60
+            Stat: Sum
+          ReturnData: false
+        - Expression: (lambdaLogCompleted / lambdaLogStarted)*100
+          Id: lambdaLogCompletePercentage
+          Label: Percentage of invocations that log a SUCCESS code against a STARTED code
+          ReturnData: false
+        - Expression: IF(lambdaLogStarted>=50, lambdaLogCompletePercentage)
+          Id: lambdaLogCompletePercentageHighTraffic
+          Label: Completion threshold calculation
+          ReturnData: true
+      AlarmActions:
+        - !Ref AlarmTopic
+      OKActions:
+        - !Ref AlarmTopic
+      Threshold: 75
+      TreatMissingData: notBreaching
+      Tags:
+        - Key: Product
+          Value: GOV.UK
+        - Key: Environment
+          Value: !Ref Environment
+        - Key: System
+          Value: Authentication
+
+  AttestationLow200ResponseProportionAlarm:
+    Type: AWS::CloudWatch::Alarm
+    DependsOn:
+      - Attestation200ResponseCountMetricFilter
+    Properties:
+      ActionsEnabled: true
```

```suggestion
      ActionsEnabled: true
```

Consider disabling/enabling the alarm based on a feature-flag.